public/css/main.css

```diff
@@ -80,54 +80,73 @@
 .title-page {
     text-align: center;
 }
-
+ 
+/*fix template index, create*/
 .btn-delete-item {
     border: none;
     background: unset;
     color: #e23434;
-    font-size: 20px;
 }
-
-.title-header {
-   display: block !important;
-   font-size: 27px !important;
+.btn-custom-option {
+    font-size: 22px;
 }
 
-.contain-btn{ 
-    float: right !important;
+.btn-option:after {
+    content: "";
+    clear:both;
 }
 
-.form-search{
-    float: right !important;
-    margin-bottom: 10px;    
+.btn-option{
+    width: 90px;
 }
-
-.btn-pencil{
-    font-size: 18px;
+.btn-option .btn-edit {
+    margin-top: 7px;
+    margin-left: 21px;
 }
 
-.col-no {
-    width: 5%;
+.btn-option a  {
+    float: left;
 }
-.col-image {
-    width: 15%;
+
+.form-delete{
+    margin-top: 3px;
 }
-.col-name {
-    width: 20%;
+
+
+/*end fix template*/
+.title-header {
+    display: block !important;
+    font-size: 27px !important;
 }
-.col-descript {
-    width: 50%;
+
+.contain-btn{ 
+    float: right !important;
+    margin-right: 1.5%;
 }
-.col-action {
-    width: 5%;
+.contain-btn.second { 
+    margin-right: 0;
+    margin-top: 20px;
 }
-.table-place{
-    table-layout: fixed;
+
+.form-search{
+    float: right !important;
+    margin-bottom: 10px;    
 }
  
 .img-place{
     max-width: 145px;
     height: 100px;
+    margin-top: 10px;
+}
+.place-descript{
+    max-height: 300px;
+    max-width: 713px;
+}
+.btn-custom {
+    margin-right: 10px;
+}
+.btn-submit{
```

Comment:
remove this, boostrap has class `pull-right` is float : right, you can using it.